chromium_src/chrome/test/base/chrome_test_launcher.cc

```diff
@@ -6,13 +6,10 @@
 #include "chrome/test/base/chrome_test_launcher.h"
 
 #include "brave/app/brave_main_delegate.h"
-#include "brave/browser/brave_content_browser_client.h"
 #include "brave/utility/brave_content_utility_client.h"
 
 #define ChromeContentUtilityClient BraveContentUtilityClient
-#define ChromeContentBrowserClient BraveContentBrowserClient
```

Comment:
this is not right because it still has `BrowserTestChromeContentBrowserClient` which subclasses `ChromeContentBrowserClient` and it needs to subclass `BraveContentBrowserClient`

Reply:
No, the upstream commit removes `BrowserTestChromeContentBrowserClient` here https://source.chromium.org/chromium/chromium/src/+/66759752273c0966338d488e7df917f9166e1b02. Unless I'm just not following you, let me know.